go/src/socialapi/Makefile

```diff
@@ -99,7 +99,7 @@ testcollaboration:
 
 testmodels:
 	@echo "$(OK_COLOR)--> model tests... $(NO_COLOR)"
-	@`which go` test -c models/*.go
+	@`which go` test -race -coverprofile=profile.out -covermode=atomic -c models/*.go
```

Comment:
Would it be possible to use package names instead of file paths?

```
@`which go` test -race -coverprofile=profile.out -covermode=atomic -c socialapi/models
```